addons/mail/static/tests/discuss/call/call_tests.js

```diff
@@ -250,3 +250,33 @@ QUnit.test("join/leave sounds are only played on main tab", async () => {
     await contains(".o-discuss-Call", { target: tab2.target, count: 0 });
     await assertSteps(["tab1 - play - channel-leave"]);
 });
+
+QUnit.test("should also invite to the call when inviting to the channel", async (assert) => {
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({
+        email: "testpartner@odoo.com",
+        name: "TestPartner",
+    });
+    pyEnv["res.users"].create({ partner_id: partnerId });
+    const channelId = pyEnv["discuss.channel"].create({
+        name: "TestChanel",
+        channel_member_ids: [Command.create({ partner_id: pyEnv.currentPartnerId })],
+        channel_type: "channel",
+    });
+    const { openDiscuss } = await start({
+        hasTimeControl: true,
+        mockRPC(route, args) {
+            if (args.method === "add_members" && args.kwargs.invite_to_rtc_call) {
```

Comment:
Is it possible/easy enough to handle this in the mock server and actually have the ringing session appear rather than checking the route param?